src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+
+    <test name="Homework18" preserve-order="false">
+        <classes>
+            <methods>
+                    <include name="playSongTest"/>
+            <class name="LoginTests"/>
```

Comment:
![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/44592718/fad747a0-f293-4e3a-bfe5-423b059f8b15)

The structure defined method here is not correct -> method playSongTest muse be in `class name="LoginTest"`
//HINT: 
```
 <class name="Homework18">
                <methods>
                    <include name="playSongTest"/>
                </methods>
   </class>
```